build/package/Dockerfile.java-toolset

```diff
@@ -0,0 +1,41 @@
+FROM registry.access.redhat.com/ubi8
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# Add adoptopenjdk repo
+COPY build/package/yum.repos.d/adoptopenjdk.repo /etc/yum.repos.d/adoptopenjdk.repo
+
+# Install Adoptopenjdk
+RUN yum install -y wget && \
+    INSTALL_PKGS="adoptopenjdk-16-hotspot" && \
+    yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
+    yum clean all -y && \
+    exactVersion=$(ls -lah /usr/lib/jvm | grep "adoptopenjdk-16-hotspot" | awk '{print $NF}' | head -1) && \
+    alternatives --set java /usr/lib/jvm/${exactVersion}/bin/java && \
+    alternatives --set javac /usr/lib/jvm/${exactVersion}/bin/javac
```

Comment:
I think `alternatives --set` is not needed. IIRC we only do this in the Jenkins agent images because they come with Java already. Doing this turns out to be very brittle. I would like to avoid it.

Reply:
'FROM adoptopenjdk/openjdk11:ubi' will simplify things

Reply:
Yes, but we should have a discussion what base images we want to use. So far we chose RedHat based ones. Let's look at this together.